core-web/package.json

```diff
@@ -74,6 +74,7 @@
         "@nx/angular": "19.5.6",
         "@tarekraafat/autocomplete.js": "^10.2.6",
         "@tinymce/tinymce-angular": "^7.0.0",
+        "@tinymce/tinymce-react": "^5.1.1",
```

Comment:
Why here?

Reply:
Our SDKs use the `node_module` from `core-web`